src/components/Icon/IconLoading/IconLoading.scss

```diff
@@ -0,0 +1,89 @@
+@import "src/styles/mixins/_icons-loading-delay";
+
+.icons-loading {
+  padding: 0;
+  margin: 0 $spaceS2;
+  list-style-type: none;
+
+  &__item {
+    display: inline-block;
+    box-sizing: border-box;
+    width: 7.059em;
+    height: 7.059em;
```

Comment:
this value are the same and i found it in another pr here: https://github.com/ovh-ux/ovh-icon-database-front/pull/31/files#r160364411, please var.